.github/workflows/linting.yml

```
@@ -0,0 +1,17 @@
+name: Run code linting
```

```suggestion
name: Run python code linting
```
I would specify python code linting (either that, or add some C++ linter).